src/main/java/de/tum/in/www1/artemis/domain/Feedback.java

```diff
@@ -215,6 +215,10 @@ public void setPositive(Boolean positive) {
         this.positive = positive;
     }
 
+    public void setPositiveViaCredits(Double credits) {
```

Comment:
Why do you have the credits as a parameter here? I think this should use the credits-attribute directly or update both attributes (credits and positive)